tests/x11/reboot_gnome.pm

```diff
@@ -24,7 +24,7 @@ sub run() {
     if (get_var("SHUTDOWN_NEEDS_AUTH")) {
         assert_screen 'reboot-auth';
         wait_still_screen;
-        type_password;
+        type_string $testapi::password, max_interval => 5;
```

Comment:
relying on that will make the timing a bit delicate. So I will add a comment - but to do that I need to get apparmor working on my instance ;(